addons/pos_loyalty/static/src/tours/PosLoyaltyTour.js

```diff
@@ -268,9 +268,12 @@ ProductScreen.do.clickPartnerButton();
 ProductScreen.do.clickCustomer('AAA Partner');
 ProductScreen.do.clickDisplayedProduct('Product B');
 ProductScreen.do.clickDisplayedProduct('Product A');
```

Comment:
```suggestion
ProductScreen.check.totalAmountIs('100.00');
ProductScreen.do.clickDisplayedProduct('Product A');
```

Instead in saas-16.3, can we start at 16.0?

Reply:
but 16.3 is where the tests started crashing, no?

Reply:
No, there are also cases linked to it starting 16.0.